python/lsst/daf/butler/formatters/parquet.py

```diff
@@ -455,7 +453,23 @@ def astropy_to_arrow(astropy_table: atable.Table) -> pa.Table:
     meta_yaml_str = "\n".join(meta_yaml)
     md[b"table_meta_yaml"] = meta_yaml_str
 
-    schema = pa.schema(type_list, metadata=md)
+    # Convert type list to fields with metadata.
+    fields = []
+    for name, pa_type in type_list:
+        field_metadata = {}
+        if description := astropy_table[name].description:
+            field_metadata["doc"] = description
+        if units := astropy_table[name].unit:
+            field_metadata["units"] = str(units)
```

Comment:
This is probably the point where we decide whether we want this to be "units" or "unit".  I may have put "units" in the drp_tasks PR that spawned this, but if astropy uses "unit" maybe we should, too.

Reply:
This is an interesting question.  Right now, our afw Fields use `doc`/`units`, which is what I thought you were going for here.  Astropy tables use `description`/`unit`, which I'm using as exact convertibles.  And right here we're defining the precedent for what should go into an arrow schema.  So I think that either we should go with the afw convention or the astropy convention.